content/akeneo-event-platform/getting-started.md

```diff
@@ -1,4 +1,161 @@
+
+    ```bash
+    client_id:secret
+    4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s:5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s
+    ```
+
+    ```makefile
+    base64 encoded
+    NGdtNHJub2l6cDhnc2tna2swODBzc29vODAwNDBnNDRrc293d2d3ODQ0azQ0c2MwMHM6NWR5dm8xejZ5MzRzbzRvZ2tna3N3ODhvb2tvb3dzMDBjZ29jNDg4a2NzOHdrNGM0MHM=
+    ```
+
+3. **Retrieve Your API Token:**
+   - After that, you can make the API call to retrieve your `API token`
+
+    ```json
+    curl --request POST http://your-host/api/oauth/v1/token \
+    --header "Content-Type: application/json" \
+    --header "Authorization: Basic YOUR_BASE_64_CLIENT_ID_AND_SECRET" \
+    --data-raw '{
+        "grant_type": "password",
+        "username": "your_API_username",
+        "password": "its_password"
+    }'
+    ```
+
+   You can check the official [token generation documentation](/documentation/authentication.html#token-generation) for further information.
+
+
+<aside>
+ℹ️ Note that the token has a lifespan of one hour.
+
+</aside>
+
+<aside>
+🛠 You can also use a custom app if you like. As long as you have a `client_id` alongside an `API token`, you are good to go for the next step
+
+</aside>
+
+### 2. Create a Subscriber
+
+After authentication, you can create a subscriber. A subscriber defines where events should be sent.
+
+```json
+curl --request POST 'https://event.prd.sdk.akeneo.cloud/api/v1/subscriber' \
+--header 'X-PIM-URL: THE_TARGET_PIM_URL' \
+--header 'X-PIM-TOKEN: YOUR_APP_OR_CONNECTION_TOKEN' \
+--header 'X-PIM-CLIENT-ID: YOUR_APP_OR_CONNECTION_CLIENT_ID' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "name": "example subscriber name",
+    "contact": {
+        "technical_email": "subscriber.test.email@example.com"
+    }
+}'
+```
+
+<aside>
+📌 You need to store the ID in the response payload. We will use it in the next step.
+
+Response Example :
+
+```json
+{
+    "id": "01905a84-a3b7-766e-a49f-5519c35fa7a0",
+    "tenant_id": "pim_tenant",
+    "client_id": "3444ec1b-058e-4208-9b6c-284f47a7aa17",
+    "name": "subscriber name",
+    "subject": "https://pim.cloud.akeneo.com",
+    "status": "active",
+    "contact": {
+        "technical_email": "subscriber.email@example.com"
+    },
+    "created_at": "2024-06-27T16:26:00.503422Z",
+    "updated_at": "2024-06-27T16:26:00.503422Z"
+},
+```
+
+</aside>
+
+### 3. Create a Subscription
+
+With a subscriber in place, the next step is to create a subscription to specify which events you want to receive.
+
+In this example, we will use an HTTPS destination (HTTPS is mandatory for secure communication).
+
+To create a subscription, you will need a destination URL.
+
+<aside>
+💡 You don’t have to worry about the secret part of the configuration for now, they are used to sign the payload (more information below in the API presentation section)
+
+</aside>
+
+```json
+curl --request POST 'https://event.prd.sdk.akeneo.cloud/api/v1/subscriber/$subscriber_Id}/subscription' \
```

Comment:
again use the "bash env var" to automatically populate the subscriber_id to be easy to use in a copy/paste way